### PR TITLE
Pre-2.0.0 release changelog update

### PR DESCRIPTION
**Pull request by mishioo**
I've also added a short desc in the readme file with link to docs, because it bothered me so much that it wasn't there.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,40 +5,66 @@ All notable changes to this project will be documented in this file.
...
-- Enum members (`EnumValue`) have attributes,
-- Support for `struct` members.
+- Basic docs.
 
-### Breaking changes
+#### Breaking changes
 
 - The generator depends on `LeanCode.Contracts` instead of `LeanCode.CQRS` package,
 - `Date`, `Time` and `DateTime` known-types are gone,
 - The tool targets both .NET 6 and .NET 7 now.
```
Comment (mishioo) [inline CHANGELOG.md:27]: Is this really a braking change?

Comment (jakubfijalkowski) [reply CHANGELOG.md:27]: Unfortunately - yes. The behavior changed slightly. :(

Comment (mishioo) [reply CHANGELOG.md:27]: Doesn't that mean that contracts v1.1.1, which also got this change, should actually be v2.0.0? :sweat_smile:

Comment (jakubfijalkowski) [reply CHANGELOG.md:27]: This only affects generator, not contracts. Plus. Contracts 1.1 were introduced with v2 of the generator (so they lag slightly).

**Review by jakubfijalkowski — APPROVED** [review]

**Pull request merged**